eureka_ml_insights/data_utils/simpleqa_utils.py

```
@@ -0,0 +1,32 @@
+import ast
+import json
+import math
+import re
+from dataclasses import dataclass
+
+import pandas as pd
+
+from .transform import DFTransformBase
+
+@dataclass
+class SimpleQA_MetadataExplode(DFTransformBase):
+    metadata_column: str
+
+    def transform(self, df: pd.DataFrame) -> pd.DataFrame:
+        df[self.metadata_column] = df[self.metadata_column].apply(lambda x: ast.literal_eval(x) if isinstance(x, str) else x)
```

If `x` is an invalid Python literal, `ast.literal_eval()` will raise an exception. Do we want to handle this gracefully here or is it okay to let the program crash in this case?